model/survey-response-logger.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+/**
+ * @module model/survey-response-logger
+ */
+
+const Sequelize = require('sequelize');
+
+/**
+ * Registers model with Sequelize
+ * @param {Sequelize} sequelize - database instance
+ * @returns {Null} nothing
+ */
+function register (sequelize) {
+    /**
+     * Each SurveyResponseLogger will be associated with a completed SurveyInstance.
+     * Each record represents the timestamps associated with when the question was answered.
+     * It also holds the timestamps for next and previous buttons pressed for each question while filling the survey.
+     * @typedef {Object} SurveyResponseLogger
+     */
+    sequelize.define(
+        'survey_response_logger',
+        {
+            ansTimestamp: {
+                type: Sequelize.DATE
+            },
+            prevTimeStamp: {
+                type: Sequelize.DATE
+            },
+            nextTimeStamp: {
+                type: Sequelize.DATE
+            }
```

Comment:
These changes are expected in the model right?
Will change the column names from the model.
Do you require these changes in the api router as well or is it ok there the way it is?